.github/workflows/antithesis-test.yml

```diff
@@ -0,0 +1,112 @@
+---
+name: Build and trigger Antithesis exploration
+
+on:
+  push:
+    branches: [main, release-3.4, release-3.5, release-3.6]
+  pull_request:
```

Comment:
This should be temporary for just experimentation when working on PR, should be removed before merging